### PR TITLE
Hedera node transaction retries

### LINKED ISSUE #643: Transfer process > timeout
Handle the process and try until success. Currently happening on mainnet.

### PR DESCRIPTION
**Pull request by vectorman1**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #643

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: config/parser/node.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ type Hedera struct {
 	Network        string            `yaml:"network"`
 	Rpc            map[string]string `yaml:"rpc"`
 	StartTimestamp int64             `yaml:"start_timestamp"`
+	MaxRetry       int               `yaml:"max_retry"`
```
Comment (failfmi) [inline config/parser/node.go:68]: Can we set a default value. Also, docs have to be updated.

**Review by svetlio8 — APPROVED** [review]

**Review by svetlio8 — COMMENTED** [review]
Please check the e2e

File: config/parser/node.go [hunk 2]
```diff
@@ -65,6 +65,7 @@ type Hedera struct {
 	Network        string            `yaml:"network"`
 	Rpc            map[string]string `yaml:"rpc"`
 	StartTimestamp int64             `yaml:"start_timestamp"`
+	MaxRetry       int               `yaml:"max_retry" default:"5"`
```
Comment (failfmi) [inline config/parser/node.go:68]: https://github.com/hashgraph/hedera-sdk-go/blob/main/transaction.go#L74 - we actually downgrade it, either switch to 10 or more

**Review by failfmi — APPROVED** [review]
Let's wait for E2E.

**Review by Psykepro — APPROVED** [review]

**Comment by failfmi** [comment]
it turns out default is not working as expected.

**Review by Psykepro — APPROVED** [review]

**Pull request merged**